state/entrySyncing.go

```diff
@@ -317,7 +320,7 @@ func (s *State) GoSyncEntries() {
 		}
 		lastfirstmissing = firstMissing
 		if firstMissing < 0 {
-			s.EntryDBHeightComplete = s.GetHighestSavedBlk()
+			s.EntryDBHeightComplete = highestSavedBlk
```

Comment:
Is the intention to log this line on every iteration? (In the old scenario like every 10ms)
If not I think this should be inside a if s.EntryBlockDBHeightComplete < highestSavedBlk {} condtion